background.js

```diff
@@ -1,27 +1,159 @@
 const pluginFlag = "pluginFlag"
+const translatedFlag = "pluginFlag"
 var shouldTranslate = false
-let initalRequestMethod = "shouldTranslate"
+let initialRequestMethod = "shouldTranslate"
 let updateRequestMethod = "updateShouldTranslate"
 let reloadRequestMethod = "reloadShouldTranslate"
-
+let startTranslateRequestMethod = "startTranslate"
+var currentTranslatingPage = [];
+var currentTabID = 0
```

Comment:
I'm not sure if we need to save it, I think we will call `chrome.tabs.query` each time when we need to do some tab-related change.

Reply:
We need it because this indicates what tab we are currently on and the previous tab. Also there is an array to save the tab that we have translated.